code/5549-kristijan-ralevski/script.js

```diff
@@ -1,3 +1,227 @@
-document.addEventListener('DOMContentLoaded', () => {
-    // Initialize the application when the DOM is fully loaded
-});
+fetch('https://raw.githubusercontent.com/sweko/internet-programming-a98db973kwl8xp1lz94kjf0bma5pez8c/refs/heads/main/data/doctor-who-episodes.json')
```

Comment:
Used like this, there is no guarantee that the DOM will be ready when the fetch finishes